### DISCUSSION TITLE
Adding a plugin into my vue 3 nuxt project.

### DISCUSSION THREAD

**Discussion by martman85** [nuxt/nuxt#8656] [Questions (Nuxt 2)] [answered]
I would like to add a plugin to my vue 3 nuxt project. Which should be pretty straight forward but I am having trouble figuring out how to do this. I looked at this page and followed its instructions:
https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-plugins

So, for example, if I want to install vueform slider (https://github.com/vueform/slider):

1) I run `npm install @vueform/slider --save`
2) I create a new file in my plugins directory called 'slider.js'. This file looks like this:
`import Vue from 'vue'
import Slider from '@vueform/slider'

Vue.use(Slider)`
3) Then I refer to the slider.js in my nuxt-config.js like this:
`plugins: [
    '@/plugins/slider'
  ],`
4) In a vue component I have a data value called 'value' and in my template I add this code:
`<Slider v-model="value" />`

This is nothing special and should work. However I get errors that keep me from using this plugin.

First of all I get these errors:

![image](https://user-images.githubusercontent.com/10497398/104891749-a0453a00-5971-11eb-9724-870b81598ac8.png)

and then it says that it's not able to find the custom Slider component.

![image](https://user-images.githubusercontent.com/10497398/104891890-ca96f780-5971-11eb-8683-e3fe919c0c60.png)

My question: What am I doing wrong? I'm following the instructions in the nuxt documentation but is this also the right way to do this in vue 3? If not, can someone point me in the right direction for this?

**Comment by TheAlexLichter** [comment, marked as answer]
Nuxt does not support Vue 3 yet ☺️ 

See #5708

**Reply by martman85** [reply]
Thanks, indeed this is the problem :)
